zipline/finance/slippage.py

```diff
@@ -18,21 +18,98 @@
 import math
 from six import with_metaclass, iteritems
 
+import numpy as np
 from pandas import isnull
 
 from zipline.finance.transaction import create_transaction
+from zipline.utils.cache import ExpiringCache
 
 SELL = 1 << 0
 BUY = 1 << 1
 STOP = 1 << 2
 LIMIT = 1 << 3
 
+SQRT_252 = math.sqrt(252)
 
-class LiquidityExceeded(Exception):
-    pass
+DEFAULT_VOLUME_SLIPPAGE_BAR_LIMIT = 0.025
+ROOT_SYMBOL_TO_ETA = {
```

Comment:
I moved these constants to their own file and added some comments and docstrings to explain them